sass/screen.scss

```diff
@@ -10,7 +10,7 @@ $FontAwesomePath: "../fonts/font-awesome/";
 @import "font-awesome";
 @include font-face(woodygoodies2nf, '../fonts/woodygoodies2nf/woodygoodies2nf-webfont');
 
-$red: #D8392F;
+$red: green;
```

Comment:
This is stupid, you should change the variable name to the right colour and make it into its own variable. what are you thinking?